cmd/containerd/command/main.go

```diff
@@ -125,6 +141,54 @@ can be used and modified as necessary as a custom configuration.`
 			}
 		}
 
+		// Add openTelemetry initialization here. Currently only stdout exporter is
+		// supported. TODO: support other exporter.
+		withOtelExporter := context.GlobalBool("with-otel-exporter")
+
+		exporter, err := stdout.NewExporter(
+			stdout.WithPrettyPrint(),
+		)
+		if err != nil {
+			logrus.Fatalf("failed to initialize stdout export pipeline: %v", err)
+		}
+
+		if !withOtelExporter {
+			exporter.Shutdown(ctx)
+		}
+
+		// Initialize Tracer provider
+		bsp := sdktrace.NewBatchSpanProcessor(exporter)
+		tp = sdktrace.NewTracerProvider(
+			sdktrace.WithSampler(sdktrace.AlwaysSample()),
```

Comment:
Do we want to sample all the time? It'd be good during development time but might be expensive in production.

Reply:
Sure it doesn't make sense. Will take care of it.